.snippets/webservers/Caddyfile

```diff
@@ -0,0 +1,56 @@
+{
+    servers :443 {
+        timeouts {
+            read_body 120s
+        }
+    }
+}
+
+<domain> {
+    root * /var/www/pterodactyl/public
+
+    file_server
+
+    php_fastcgi unix//run/php/php8.1-fpm.sock {
```

Comment:
Sorry, never used Caddy, is // intentional?

Reply:
Yes, it is intentional. I have it deployed like that for 2+ years, works just fine.